docs/NOTES.md

```diff
@@ -0,0 +1,64 @@
+# Docs process
+
+Pants currently hosts documentation at Readme.com, and we use a combination of their `rdme` tool to sync handwritten markdown docs, and a custom `generate-docs` script to update Pants' reference documentation.
+
+Currently the rdme process is manual, until we bed down the process, at which point we'll add it to CI.
+
+The motivation for in-repo docs is covered [on this Google doc](https://docs.google.com/document/d/1bZE8PlF9oRzcPQz4-JUFr5vfD0LFHH4V3Nj2k221CFM/view)
+
```

Comment:
Any way to "preview" changes locally?

Reply:
Not by using Readme, unfortunately. We've identified future work in migrating to Sphinx, which will allow you to generate your own local docs, but that's not possible now.

Given the docs are common markdown, most markdown rendering tools should look fine. If you want to see it on readme's site, it should be possible for you to create your own scratch readme account and sync the docs up to that. The tools are reasonably self-contained.